src/core/tests/test_app.py

```diff
@@ -64,6 +64,10 @@ def test_app_exit(self):
         self.app.exit()
 
         self.assertActionPerformed(self.app, 'exit')
+    
+    def test_app_is_full_screen(self):
+        self.app.set_full_screen(self.app.main_window)
+        self.assertEqual(self.app.is_full_screen, True)    
```

Comment:
At column 56: (W291) trailing whitespace

Reply:
Trailing whitespace removed
```suggestion
        self.assertEqual(self.app.is_full_screen, True)    
```